includes/class-wcpdf-install.php

```diff
@@ -189,6 +189,10 @@ protected function install() {
 
 		// set transient for wizard notification
 		set_transient( 'wpo_wcpdf_new_install', 'yes', DAY_IN_SECONDS * 2 );
+
+		// schedule the invoice yearly reset number action
+		$invoice = wcpdf_get_invoice( null );
```

Comment:
this should not require the invoice object, it's unrelated to the document itself